WebApp/src/components/BasicTest/BasicTestContainer.tsx

```diff
@@ -0,0 +1,30 @@
+import './BasicTestContainer.css';
+import { IonText, IonButton } from '@ionic/react';
+import React from 'react';
+
+interface ContainerProps {}
+
+const ExploreContainer: React.FC<ContainerProps> = () => {
+  const quotes = [
+    'Georges is the best programmer',
+    'Etien is the best programmer',
+  ];
+
+  const [quote, setQuote] = React.useState(quotes[0]);
```

Comment:
Puisque c'est la lib standard je préfère utiliser les fonctions directement. Ça allège le code.
```suggestion
const [quote, setQuote] = useState(quotes[0]);
```

Reply:
oopsii